### PR TITLE
Working get review by ID route, passing unit test

### PR DESCRIPTION
**Pull request by janinebar**

### CODE REVIEW

**Review by ngynandrea — COMMENTED** [review]
Looks good!

**Review by ngynandrea — APPROVED** [review]

File: routes/reviews.go [hunk 1]
```diff
@@ -191,3 +191,32 @@ func GetReviewsHandler(w http.ResponseWriter, r *http.Request) {
...
 	json.NewEncoder(w).Encode(reviews)
 }
+
+func GetReviewHandler(w http.ResponseWriter, r *http.Request) {
+
+	// Get review ID from params
+	params := mux.Vars(r)
+	reviewID, reviewIDExists := params["reviewID"]
+
+	log.Printf("%v", params)
```
Comment (kftang) [inline routes/reviews.go]: remove this

File: server.go [hunk 2]
```diff
@@ -23,6 +23,7 @@ func main() {
 	router.HandleFunc("/foodtrucks", routes.GetFoodTrucksHandler).Methods("GET")
 	router.HandleFunc("/foodtrucks/{foodTruckID}", routes.GetFoodTruckHandler).Methods("GET")
 	router.HandleFunc("/reviews", routes.GetReviewsHandler).Methods("GET")
+	router.HandleFunc("/review/{reviewID}", routes.GetReviewHandler).Methods("GET")
```
Comment (kftang) [inline server.go]: reviews with an s

File: routes/reviews.go [hunk 3]
```diff
@@ -191,3 +191,32 @@ func GetReviewsHandler(w http.ResponseWriter, r *http.Request) {
...
+	if err != nil {
+		w.WriteHeader(http.StatusInternalServerError)
+		log.Printf("ERROR: %v", err)
+		return
+	}
+
+	// Send response
+	w.Header().Set("Content-Type", "application/json")
+	json.NewEncoder(w).Encode(review)
+	w.WriteHeader(http.StatusOK)
```
Comment (kftang) [inline routes/reviews.go]: don't need the last line, i think it breaks the content-type

**Review by kftang — CHANGES_REQUESTED** [review]

**Review by kftang — APPROVED** [review]

**Pull request merged**